sample_apps/simple_app_python/example_app.py

```diff
@@ -23,16 +25,34 @@
 # Script defaults
 ATTEST_KEY_FILE                 = 'attest_key_file.bin'
 APP_DATA_DIR                    = './app1_data/'
+PROVISIONING_DIR                = './provisioning'
 EXAMPLE_MEASUREMENT             = 'example_app.measurement'
 APP_OP_TYPES                    = 'cold-init, get-certified, run-app-as-client, run-app-as-server'
 PLATFORM_ATTEST_ENDORSEMENT     = 'platform_attest_endorsement.bin'
 PLATFORM_CERT_FILE              = 'platform_file.bin'
 POLICY_STORE                    = 'store.bin'
 POLICY_HOST                     = 'localhost'
 POLICY_HOST_PORT                = 8123
-SERVER_APP_HOST                 = 'localhost'
+SERVER_APP_HOST                 = '127.0.0.1'
 SERVER_APP_PORT                 = 8124
 
+# ------------------------------------------------------------------------------
+# Script global symbols to app-data dirs. When this program is invoked by the
+# driver script, run_example.sh, these names are used to specify the --data-dir
+# for the client v/s the server.
+CLIENT_APP_DATA='app1_data'
+SERVER_APP_DATA='app2_data'
+
+# Basenames for output files created to persist certificates and keys
+# Names are exactly those used in test_client_server_mtls.py, based on
+# which the client-server mTLS SSL secure-channel code in this program
+# is modelled.
+SERVER_ROOT_SIGNED_CERT = 'server.PEM.cert'
```

Comment:
This naming is a bit confusing. When some application obtains a cert from the CS, it can use it for both server and client authentication. In that respect, `APP_CERT` and `APP_KEY` stored in the corresponding app dir would be better IMO.